docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -337,71 +373,212 @@ apiVersion: "observability.openshift.io/v1"
 kind: ClusterLogForwarder
 spec:
   filters:
-  - name: detectexception
-    type: detectMultilineException
-  - name: parse-json
-    type: parse
-  - name: labels
-    type: openShiftLables
-    openShiftLabels:
-      foo: bar
+    - name: my-multiline
+      type: detectMultilineException
+    - name: my-parse
+      type: parse
+    - name: my-labels
+      type: openShiftLabels
+      openShiftLabels:
+        foo: bar
   pipelines:
-  - name: application-logs
-    filterRefs:
-    - detectexception
-    - labels
-    - parse-json
+    - name: app-logs
+      filterRefs:
+      - my-multiline
+      - my-parse
+      - my-labels
+----
+NOTE: Drop filter, Prune filter and KubeAPIAudit filters remain unchanged
+[source, yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  filters:
+    - name: drop-debug-logs
+      type: drop
+      drop:
+      - test:
+        - field: .level
+          matches: debug
+    - name: prune-fields
+      type: prune
+      prune:
+        in:
+        - .kubernetes.labels.foobar
+        notIn:
+        - .message
+    - name: audit-logs
+      type: kubeAPIAudit
```

Comment:
This may change with our standards PR change?

Reply:
Yes,  but needs to merge